### ISSUE TITLE
Game Libary Custom Games

### ISSUE THREAD

**Issue by TuRagChoi** [Vencord/plugin-requests#107]
### Plugin Description

My Idea is, that there is a button in the old discord game libary where you can add custom games / programs to your libary so that you don't need to open an extra launcher like steam or epic games to start your games
(the libary automatically unlocks itself, after activating the plugin)

reference picture: https://imgur.com/a/T59Z8OV

### Existing Plugin for other mod

-

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Sqaaakoi** [comment]
why don't you use your OS/DE's built in start menu or whatever as a launcher? also you do know those game launchers will probably have to be running anyway to provide some services for the game to run (for example; steam networking, achievements)

**Comment by TuRagChoi** [comment]
> why don't you use your OS/DE's built in start menu or whatever as a launcher? also you do know those game launchers will probably have to be running anyway to provide some services for the game to run (for example; steam networking, achievements)

Yes I know, but it is more is easier to use if you have discord open anyway on a second monitor
